src/test/scala/ApplicationSpec.scala

```diff
@@ -17,17 +17,27 @@ class ApplicationSpec extends AnyFlatSpec with Matchers {
     val xmlTestURL = getClass.getResource("/GradeSchool_successful.xml")
     val outputFileURL = getClass.getResource("/outputs/output.txt")
     val exercismOutput: JSONObject = Application.toExercismJSON(xmlTestURL, outputFileURL)
+
     assert(exercismOutput.getInt("version") == 2)
     assert(exercismOutput.getString("status") == "pass")
-    assert(exercismOutput.getString("message") == "")
-    assert(exercismOutput.get("tests").asInstanceOf[Array[JSONObject]].length == 7)
+    assert(exercismOutput.opt("message") == None)
+
+    val testCases = exercismOutput.get("tests").asInstanceOf[Array[JSONObject]]
+    assert(testCases.length == 7)
+    assert(testCases(0).toString() == """{"output":"TOIMPLEMENT","name":"empty school","test_code":"TOIMPLEMENT","message":"None","status":"pass"}""")
```

Comment:
It looks like `"message":"None"` is not what you want, or is it?

Reply:
Indeed it wasn't :)